doctr/models/recognition/sar.py

```diff
@@ -125,13 +125,13 @@ def call(
         # run first step of lstm
         # holistic: shape (N, rnn_units)
         _, states = self.lstm_decoder(holistic, states, **kwargs)
-        sos_symbol = self.vocab_size + 1
+        sos_symbol = self.vocab_size
```

Comment:
The vocab_size here is unchanged compared to earlier. If I understand well, this points to the index of the last symbol ("<eos>") of the vocab. If that is so, I believe there is a typo "sos_symbol"
Additionally, for clarity, this is an index, so I suggest naming it `eos_idx`